### DISCUSSION TITLE
Append Tag to download path

### DISCUSSION THREAD

**Discussion by charsi** [jesec/flood#442] [Ideas]
With ruTorrent gui I was able to set download location automatically based on tags. 
So, things labelled with 'MOVIES' automatically got moved to `/mnt/media/MOVIES` on completion. (`/mnt/media` being the default path)

This is what the option looks like in ruTorrent - 
![image](https://user-images.githubusercontent.com/826310/140021720-9c4b0ea0-e1b1-4a60-94e1-e91daa6d74b7.png)

It'll be great if Flood also got this feature.

**Comment by jesec** [comment]
Flood memorizes the last destination used, for each tag.

**Comment by charsi** [comment]
Doesn't seem to be working for me. 
I tried adding a tag after setting the location and also setting both tag and location when adding the first torrent. Also tried adding from the ui and the api. In all cases the subsequent torrent kept the base location. -


![image](https://user-images.githubusercontent.com/826310/140024761-ea9931e2-70d3-4216-8e95-99692321b8ae.png)

**Reply by jesec** [reply]
Once you add a torrent with a tag and a destination, next time when you add a torrent, Flood automatically fills the same destination for you after you choose the tag. 

Flood does this for each tag. You should simply add the torrent with the tag and the desired destination (in your case, `/data/TV SERIES/`).

**Reply by charsi** [reply]
Got it to work in the end. I had to first set the default location to empty in the settings.

**Comment by charsi** [comment]
Also I sometimes manually move tagged torrents to their 2nd level subfolders. For example /data/TV SERIES/some_series/season01. I wouldn't really want the next torrent to go there if it is a different series.

**Reply by charsi** [reply]
Answering myself.. Flood only remembers the location as set when initially adding the torrent. So if I later change the location for a tagged torrent the subsequent torrents still get the original location (with the first level sub-folder, not the second). Which is how I want it to work.
